src/components/TopBar/TopBar.vue

```diff
@@ -68,36 +75,54 @@
 			:is-sidebar="isSidebar"
 			:model="localMediaModel" />
 
-		<div class="top-bar__buttons">
-			<CallButton class="top-bar__button" />
+		<CallButton class="top-bar__button" />
 
+		<template v-if="showOpenSidebarButton">
 			<!-- sidebar toggle -->
-			<NcActions v-if="showOpenSidebarButton"
+			<NcButton v-if="!isInCall"
+				class="top-bar__button"
+				close-after-click="true"
+				type="tertiary"
+				@click="openSidebar">
+				<template #icon>
+					<MenuIcon :size="20" />
+				</template>
+			</NcButton>
+
+			<!-- chat button -->
+			<div v-if="isInCall"
+				class="chat-button">
+				<NcActions class="top-bar__button"
+					close-after-click="true"
+					:container="container">
+					<NcActionButton key="openSideBarButtonMessageText"
+						@click="openSidebar('chat')">
+						<template #icon>
+							<MessageText :size="20"
+								fill-color="#ffffff" />
+						</template>
+					</NcActionButton>
+				</NcActions>
+				<NcCounterBubble v-if="!isSidebar && isInCall && unreadMessagesCounter > 0"
+					class="chat-button__unread-messages-counter"
+					:highlighted="hasUnreadMentions">
+					{{ unreadMessagesCounter }}
+				</NcCounterBubble>
+			</div>
+
+			<!-- participants button -->
+			<NcButton v-if="isInCall & !isOneToOneConversation"
```

Comment:
I'm pretty sure you don't want to do bitflag comparisions...
```suggestion
			<NcButton v-if="isInCall && !isOneToOneConversation"
```